test/kouta_indeksoija_service/fixture/common_indexer_fixture.clj

```diff
@@ -109,6 +116,10 @@
   [search-index oid key expected]
   (count (filter-hits-by-key search-index oid key expected)))
 
+(defn hit-key-not-empty
+  [search-index oid key]
+  (not (nil? (seq (some #(get % key) (:hits (get-doc search-index oid)))))))
```

Comment:
Not nil on käsittääkseni sama kuin some? Ja kai tässä vois (fn) käyttää kun sitä enimmäkseen käytetty?

```suggestion
  (some? (seq (some (fn [h] (get h key)) (:hits (get-doc search-index oid))))))
```